### PR TITLE
Fix problem with year logic getting used without a year

### PR DESCRIPTION
**Pull request by Razzeee**
@enen92 Can you please have a quick look?

### CODE REVIEW

**Comment by Razzeee** [comment]
Fix is based on http://paste.kodi.tv/edenaxigoc

**Review by enen92 — APPROVED** [review]

**Comment by enen92** [comment]
Looks ok to me, I don't have permissions to merge this though

**Comment by Razzeee** [comment]
I've added the team and gave maintain rights.

File: resources/lib/series.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def search_series(title, settings, year=None) -> None:
     log(f'Search results {search_results}')
 
     possible_matches = _match_by_year(
-        search_results, year, title) if year is not None else _filter_exact_matches(search_results, title)
+        search_results, year, title) if year is not None and len(year) > 0 else _filter_exact_matches(search_results, title)
```
Comment (romanvm) [inline resources/lib/series.py:26]: `if year` should be enough.

**Pull request merged**